src/components/board/SharingTabSidebard.vue

```diff
@@ -33,24 +33,25 @@
 				<label for="checkbox">{{ t('deck', 'Manage') }}</label>
 
 				<button v-tooltip="t('deck', 'Delete')" class="icon-delete" @click="clickDeleteAcl(acl)" />
-
 			</li>
 		</ul>
-		<CollaborationView />
+
+		<collection-list v-if="board.id" :id="&quot;board.id&quot;" :name="board.title"
```

Comment:
```suggestion
		<collection-list v-if="board.id" :id="`${board.id}`" :name="board.title"
```

Reply:
What das this means? Where can I read more about this?

Reply:
We require a string as an id of the collections list. This notation is basically a string cast, done by using template literals: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals